code/game/objects/items/toys.dm

```diff
@@ -823,6 +853,48 @@
 	desc = "Harvesters tend to have a bad habit of violently stabbing anyone they meet"
 	icon_state = "harvester"
 
+/obj/item/toy/gasha/narnar
+	name = "toy Nar-Sie"
+	desc = "The father figure to all of his faithful, the Geometer of Blood himself; NAR-SIE!"
+	icon_state = "narnar"
+
+/obj/item/toy/gasha/quote
+	name = "Robot"
+	desc = "It's a small robot toy"
+	icon_state = "quote"
+
+/obj/item/toy/gasha/quote/curly
+	icon_state = "curly"
+
+/obj/item/toy/gasha/mario
+	name = "toy plumber"
+	desc = "It's a toy of a popular plumber character.
+	icon_state = "mario"
+
+/obj/item/toy/gasha/mario/luigi
+	icon_state = "luigi"
+
+/obj/item/toy/gasha/mario/star
+	icon_state = "star"

```

Comment:
This one is inheriting mario's desc, and it's not really a plumber character, is it?